Firestore/tests/System/Admin/V1/FirestoreAdminClientSmokeTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Firestore\Tests\System\Admin\V1;
+
+use Google\ApiCore\PagedListResponse;
+use Google\Cloud\Firestore\Tests\System\FirestoreTestCase;
+use Google\Cloud\Firestore\Admin\V1\FirestoreAdminClient;
+use Google\Cloud\Firestore\FirestoreClient;
+
+/**
+ * @group firestore
+ * @group firestore-admin
```

Comment:
nit: other gapic system tests add `@group gapic`. Can we do that here?

Reply:
Done!